en/peripherals/esc_motors.md

```diff
@@ -1,12 +1,80 @@
 # ESCs & Motors
 
-Many PX4 drones use brushless motors that are driven by the flight controller via an Electronic Speed Controller (ESC) (the ESC converts a signal from the flight controller to an appropriate level of power delivered to the motor).
+Many PX4 drones use brushless motors that are driven by the flight controller via an Electronic Speed Controller (ESC).
+The ESC takes a signal from the flight controller and uses it to set control the level of power delivered to the motor.
 
-PX4 supports [ESCs that take a PWM input](../peripherals/pwm_escs_and_servo.md), ESCs that use the ESC *OneShot* standard, [DShot](../peripherals/dshot.md), [UAVCAN ESCs](../peripherals/uavcan_escs.md), PCA9685 ESC (via I2C), and some UART ESCs (from Yuneec).
+PX4 supports a number of common protocols for sending the signals to ESCs: [PWM ESCs](../peripherals/pwm_escs_and_servo.md), [OneShot ESCs](../peripherals/oneshot.md), [DShot ESCs](../peripherals/dshot.md), [UAVCAN ESCs](../peripherals/uavcan_escs.md), PCA9685 ESC (via I2C), and some UART ESCs (from Yuneec).
 
 For more information see:
 * [PWM ESCs and Servos](../peripherals/pwm_escs_and_servo.md)
+* [OneShot ESCs and Servos](../peripherals/oneshot.md)
 * [DShot](../peripherals/dshot.md)
 * [UAVCAN ESCs](../peripherals/uavcan_escs.md)
 * [ESC Calibration](../advanced_config/esc_calibration.md)
 * [ESC Firmware and Protocols Overview](https://oscarliang.com/esc-firmware-protocols/) (oscarliang.com)
+
+A high level overview of the main ESC/Servo protocols supported by PX4 is given below.
+
+## ESC Protocols
+
+### PWM
+
+[PWM ESCs](../peripherals/pwm_escs_and_servo.md) are commonly used for fixed wing vehicles and ground vehicles (vehicles that require a higher response rate like multicopters typically use oneshot or dshot ESCs).
+
+PWM ESCs communicate using a periodic pulse, where the _width_ of the pulse indicates the desired power level.
+The pulse wdith typically ranges between 1000uS for zero power and 2000uS for full power.
+The periodic frame rate of the signal depends on the capability of the ESC, and commonly ranges between 50Hz and 490 Hz (the theoretical maximum being 500Hz for a very small "off" cycle).
+A higher rate is better, in particular where a rapid response to setpoint changes is needed.
```

Comment:
```suggestion
```suggestion
A higher rate is better for ESCs, in particular where a rapid response to setpoint changes is needed.
For PWM servos 50Hz is usually sufficient, and many don't support higher rates.
``` 
```